image.yaml

```diff
@@ -27,11 +27,11 @@ packages:
       install:
           - gettext
 artifacts:
-    - url: https://maven.repository.redhat.com/ga/org/jboss/teiid/teiid-jdbc/8.12.12.6_3-redhat-1/teiid-jdbc-8.12.12.6_3-redhat-1.jar
-      md5: 6c799d9cfc66541f9fce920fd29bf7cb
-    - url: https://maven.repository.redhat.com/ga/org/jboss/teiid/teiid-hibernate-dialect/8.12.12.6_3-redhat-1/teiid-hibernate-dialect-8.12.12.6_3-redhat-1.jar
-      md5: 44833dd4953d56cb1f75f3aee0743bc9
+    - url: http://download-node-02.eng.bos.redhat.com/brewroot/repos/jb-dv-6.4-build/latest/maven/org/jboss/teiid/teiid-jdbc/8.12.11.6_4-redhat-64-6/teiid-jdbc-8.12.11.6_4-redhat-64-6.jar
+      md5: f2476af52e9dd34f5e138f4607c14b9c
+    - url: http://download-node-02.eng.bos.redhat.com/brewroot/repos/jb-dv-6.4-build/latest/maven/org/jboss/teiid/teiid-hibernate-dialect/8.12.11.6_4-redhat-64-6/teiid-hibernate-dialect-8.12.11.6_4-redhat-64-6.jar
+      md5: b6093cf75fec55bcc369b244b74e103a
 osbs:
       repository:
             name: jboss-datavirt-6-docker
-            branch: ce-1.2-openshift-driver-datavirt-6.3-jdk-8-rhel-7
+            branch: ce-1.2-openshift-driver-datavirt-6.4-jdk-8-rhel-7
```

Comment:
Branch should be in stream `1.0`: `ce-1.0-openshift-driver-datavirt-6.4-jdk-8-rhel-7`